File: webdav/ocaml/create/webdav_create.ml
(* ------------------------------------------------------------------- *)
(*
	webdav_create.ml

					Feb/13/2012

*)
(* ------------------------------------------------------------------- *)
open	Printf
open	Json_type
open	Curl_manipulate
(* ------------------------------------------------------------------- *)

let city_xx = Object ["t1351",Object ["name", String "八王子";
                   "population", Int 51274; "date_mod", String "1995-3-21"];
		"t1352",Object ["name", String "立川";
                   "population", Int 58362; "date_mod", String "1995-7-30"];
		"t1353",Object ["name", String "武蔵野";
                   "population", Int 82476; "date_mod", String "1995-5-12"];
		"t1354", Object ["name", String "三鷹";
                   "population", Int 72369; "date_mod", String "1995-8-25"];
		"t1355", Object ["name", String "青梅";
                   "population", Int 42953; "date_mod", String "1995-9-18"];
		"t1356", Object ["name", String "府中";
                   "population", Int 72648; "date_mod", String "1995-2-8"];
		"t1357", Object ["name", String "昭島";
                   "population", Int 62846; "date_mod", String "1995-4-14"];
		"t1358", Object ["name", String "調布";
                   "population", Int 25719; "date_mod", String "1995-9-23"];
		"t1359", Object ["name", String "町田";
                   "population", Int 52987; "date_mod", String "1995-12-18"]]

(* ------------------------------------------------------------------- *)
let _ = print_endline "*** 開始 ***"
let url_target = "http://host_dbase:3004/city/tokyo.json"

let str_json = Json_io.string_of_json  city_xx

let _ =
	let r,c = put url_target str_json in
		printf "%d -> %s\n" r c


let _ = print_endline "*** 終了 ***"
(* ------------------------------------------------------------------- *)

File: webdav/ruby/create/webdav_create.rb
#! /usr/bin/ruby
# -*- coding: utf-8 -*-
#
#	webdav_create.rb
#
#					Mar/18/2013
#
require 'rubygems'
require 'json'
require 'restclient'
#
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/ruby_common/net_manipulate.rb'
#
#
# ---------------------------------------------------------------------
def prepare_data_proc ()
dict_aa={}

dict_aa=dict_append_proc(dict_aa,"t1351","八王子",39287,"2006-10-19")
dict_aa=dict_append_proc(dict_aa,"t1352","立川",23864,"2006-7-27")
dict_aa=dict_append_proc(dict_aa,"t1353","武蔵野",45972,"2006-2-8")
dict_aa=dict_append_proc(dict_aa,"t1354","三鷹",78623,"2006-1-22")
dict_aa=dict_append_proc(dict_aa,"t1355","青梅",41389,"2006-5-11")
dict_aa=dict_append_proc(dict_aa,"t1356","府中",16925,"2006-9-17")
dict_aa=dict_append_proc(dict_aa,"t1357","昭島",36972,"2006-8-18")
dict_aa=dict_append_proc(dict_aa,"t1358","調布",92147,"2006-8-15")
dict_aa=dict_append_proc(dict_aa,"t1359","町田",86523,"2006-10-21")


end
# ---------------------------------------------------------------------
puts "*** 開始 ***"
#
dict_aa=prepare_data_proc()
#
json_str = JSON.pretty_generate(dict_aa)
#
url_target='http://cddn007:3004/city/tokyo.json'
RestClient.put url_target, json_str
#
puts	"*** 終了 ***"
#
# --------------------------------------------------------------------------
